.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ env:
   # Path to where test results will be saved.
   TEST_RESULTS: /tmp/test-results
   # Default minimum version of Go to support.
-  DEFAULT_GO_VERSION: 1.19
+  DEFAULT_GO_VERSION: "1.20"
```

Comment:
should we update our go.mod files to 1.20 as well?

Reply:
done